code/MANAGER/MANAGER.py

```diff
@@ -120,7 +137,7 @@ def executeMachineStatus(self) -> None:
             #Flash original program
             if self.__status is TERM_Option_e.FLASH_ORIG:
                 log.info(f"Flashing original program")
-                self.__stm.flashUC(binary_name = "STM32_org.bin")
+                self.__stm.flashUC(binary_name = 'STM32_orig.bin')

```

Comment:
Add function on term to print error message. TERM_c.printError(status, message)
`Error raise on status: {status}. Message: {message}`

Reply:
Done.